master/pkg/model/task_config_policy.go

```diff
@@ -1,34 +1,29 @@
 package model
 
 import (
-	"strings"
 	"time"
 
 	"github.com/uptrace/bun"
 
 	"github.com/determined-ai/determined/master/pkg/schemas/expconf"
-	"github.com/determined-ai/determined/proto/pkg/configpolicyv1"
 )
 
-// WorkloadType is the type of workload (experiment or NTSC) that the task config policy applies to.
-type WorkloadType string
```

Comment:
This was a great idea. Just a little ahead of its time, perhaps. Thanks for keeping the codebase clean!